### PR TITLE
BaSD-MS-50-Sessions: Redux

### PR DESCRIPTION
**Pull request by ignacioaristo**
Added the logic of Redux and Redux thunk to handle Sessions

### CODE REVIEW

File: src/redux/sessions/thunks.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    },
+    body: JSON.stringify({
+      postulant: data.postulant,
+      counselor: data.counselor,
+      date: data.date,
+      time: data.time,
+      accomplished: data.accomplished
+    })
+  };
+  console.log(data, 'data');
```
Comment (aquilesmariani) [inline src/redux/sessions/thunks.js:60]: remove console.logs

File: src/redux/sessions/thunks.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  deleteSessionFetching,
+  deleteSessionFulfilled,
+  deleteSessionRejected
+} from './actions';
+
+const URL = process.env.REACT_APP_API;
+
+export const getSessions = () => (dispatch) => {
+  dispatch(getSessionsFetching());
+  fetch(`${URL}/sessions/`)
```
Comment (aquilesmariani) [inline src/redux/sessions/thunks.js:23]:
```suggestion
  fetch(`${URL}/sessions`)
```

File: src/Components/Sessions/index.js [hunk 3]
```diff
@@ -117,13 +91,13 @@ function Sessions() {
               onClick={() => (window.location.href = `sessions/form?id=${session._id}`)}
               className={styles.sessionRow}
             >
-              <td>{`${session.postulantId?.firstName} ${session.postulantId?.lastName}`}</td>
-              <td>{session.counselorId?.firstName}</td>
-              <td>{session.date}</td>
+              <td>{session.postulantId}</td>
```
Comment (aquilesmariani) [inline src/Components/Sessions/index.js:94]: `postulantId` seems to be an object and you can't place it this way, did you forget to commit something to the server or just need to show in here the actual data?

**Review by aquilesmariani — CHANGES_REQUESTED** [review]

File: src/redux/sessions/thunks.js (hunk 2 above)
Comment (ignacioaristo) [reply src/redux/sessions/thunks.js:23]: Changed it

File: src/redux/sessions/thunks.js (hunk 1 above)
Comment (ignacioaristo) [reply src/redux/sessions/thunks.js:60]: Removed it.

File: src/Components/Sessions/index.js (hunk 3 above)
Comment (ignacioaristo) [reply src/Components/Sessions/index.js:94]: Fixed it, i need to show the data inside, example, `postulantId.firstName`

**Review by aquilesmariani — APPROVED** [review]
🚀

**Review by leimeterjoaquin — APPROVED** [review]

**Pull request merged**